ios/sdk/src/ExternalAPI.m

```diff
@@ -76,6 +82,10 @@ + (BOOL)requiresMainQueueSetup {
     if (!delegate) {
         return;
     }
+    
+    if ([name  isEqual: @"PARTICIPANTS_INFO_RETRIEVED"]) {
+        [self onParticipantsInfoRetrieved: data];
```

Comment:
Don't we want to return here?

Reply:
I was thinking to replicate one-on-one the Android behaviour, so besides calling the completionBlock, the method on the delegate is also called.

Reply:
But this is not an event per se, it's an internal one which we use for calling the completion blocks, so I'd say let's not call any listener method.